py3-sacrebleu.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: py3-sacrebleu
+  version: 2.3.0
+  epoch: 0
+  description: Reference BLEU implementation that auto-downloads test sets and reports a version string to facilitate cross-lab comparisons
+  copyright:
+    - license: BSD
```

Comment:
Git repo says `Apache-2.0`, any reason why we use BSD instead?

Reply:
Ah, copy-paste issue, good catch. Fixed! Will careful next time.